src/lib/telemetry/TelemetryReporter.ts

```diff
@@ -1,13 +1,20 @@
+import { readdir, readJson, remove, writeJson, ensureDir, ensureFile } from 'fs-extra'
+import { randomBytes } from 'crypto'
+import * as lockfile from 'lockfile'
+
 import { TelemetryClient } from '../../clients/telemetryClient'
 import { region } from '../../env'
 import userAgent from '../../user-agent'
 import { createIOContext, createTelemetryClient } from '../clients'
 import { SessionManager } from '../session/SessionManager'
 import { TelemetryLocalStore } from './TelemetryStore'
+import { ErrorReport } from '../error/ErrorReport'
+import { join, resolve } from 'path'
 
 export class TelemetryReporter {
   private static readonly RETRIES = 3
   private static readonly TIMEOUT = 30 * 1000
+  public static readonly ERRORS_DIR = resolve('~/.config/configstore/vtex/errors')
```

Comment:
Hmmmm, I'm wondering about if the errors dir should be an option this script receives, or if it should be:
```ts
public static readonly ERRORS_DIR = join(TelemetryCollector.LOCAL_TELEMETRY_DIR, 'errors')
```
Doing in a way it receives from the arguments and the `TelemetryReporter` class has in its contructor something like:
```
constructor(private errorsDir: string) 
```
Would make testing this class easier, I think, because we could easily mock an `errorDir` that we want for testing.

Reply:
Actually this wouldn't be necessary, since `TelemetryReporter` is not responsible in the moment for writing errors